README.md

```diff
@@ -3,202 +3,212 @@
-    as the `JobTrigger` is activated.
+This I/O presentation has more information on why background services are harmful and what you can
+do about them:
 
--   A `RetryStrategy` that specifies how failures should be handled. The default
-    is to handle failures using an exponential backoff strategy.
+[![Android battery and memory optimizations][io-video-img]][io-video-link]
 
--   A `lifetime` that specifies how long the Job should remain scheduled. The
-    default is to keep it until the next boot.
+There's more information on upcoming changes to Android's approach to background services on the
+[Android developer preview page][n-preview-bg-optimizations].
 
--   An optional `Bundle` of user-supplied extras. The default is an empty
-    Bundle.
+[n-preview-bg-optimizations]: https://developer.android.com/preview/features/background-optimization.html
+[io-video-img]: http://img.youtube.com/vi/VC2Hlb22mZM/hqdefault.jpg
+[io-video-link]: https://youtu.be/VC2Hlb22mZM
+[js-network-type]: https://developer.android.com/reference/android/app/job/JobInfo.Builder.html#setRequiredNetworkType(int)
+[js-charging-state]: https://developer.android.com/reference/android/app/job/JobInfo.Builder.html#setRequiresCharging(boolean)
+[js-idle-state]: https://developer.android.com/reference/android/app/job/JobInfo.Builder.html#setRequiresDeviceIdle(boolean)
 
--   A boolean indicating whether the Job should repeat. The default is false
-    (i.e. the Job will be executed once).
+### Requirements
 
--   A boolean indicating whether the Job should replace any pre-existing Job
-    with the same tag. The default is false.
+The FirebaseJobDispatcher currently relies on the scheduling component in Google Play services.
+Because of that, it won't work on environments without Google Play services installed.
 
-### Driver
+### Comparison to other libraries
 
-`Driver` is an interface that represents a component that can schedule, cancel,
-and execute Jobs. The only bundled Driver is the `GooglePlayDriver`, which
-relies on the scheduler built-in to Google Play services.
+| Library                    | Minimum API | Requires Google Play   | Service API<sup>[1](#fn1)</sup> | Custom retry strategies |
+| -------------------------- | ----------- | ---------------------- | ------------------------------- | ----------------------- |
+| Framework [JobScheduler][] | 21          | No                     | JobScheduler                    | Yes                     |
+| Firebase JobDispatcher     | 9           | Yes                    | JobScheduler                    | Yes                     |
+| [evernote/android-job][]   | 9 / 14      | No<sup>[2](#fn2)</sup> | Custom                          | Yes<sup>[3](#fn3)</sup> |
 
-### Trigger
+<a name="fn1">1</a>: Refers to the methods that need to be implemented in the Service subclass.  
+<a name="fn2">2</a>: Uses AlarmManager to support API levels >= 14 if Google Play services is unavailable  
```

Comment:
That's wrong, it should be "Uses AlarmManager to support API levels < 21 if Google Play services is unavailable.